src/components/ThumbsFeedback/ThumbsFeedback.tsx

```diff
@@ -0,0 +1,165 @@
+import styles from './ThumbsFeedback.module.css';
+import React, { FormEvent, useState, useEffect } from "react";
+import { useLocation } from '@docusaurus/router';
+import Icon from "../Icon/Icon";
+import Button from "../Button/Button";
+import { GitHubIssueLink } from "@site/src/components/GitHubIssueLink";
+import { trackEvent } from '@site/src/utils/analytics';
+import { isValidCommentLength, containsPII } from '@site/src/utils/validations';
+
+const MAX_COMMENT_LENGTH: number = 100;
+
+const ThumbsFeedback = (): JSX.Element => {
+  const [feedback, setFeedback] = useState<string | null>(null);
+  const [comment, setComment] = useState<string>("");
+  const [showButtons, setShowButtons] = useState<boolean>(true);
+  const [isSubmitted, setIsSubmitted] = useState<boolean>(false);
+  const [isSidebarScrollable, setIsSidebarScrollable] = useState<boolean>(true);
+  const location = useLocation();
+
+  useEffect(() => {
+    setShowButtons(true);
+    setIsSubmitted(false);
+    setFeedback(null);
+    setComment("");
+  }, [location.pathname]);
+
+  // Currently feedback will be hidden if sidebar is scrollable AND within 350 of the bottom as this is the height when the textarea is active
+  // Can plan to implement this better going forward
+  useEffect(() => {
+    const checkSidebarScrollable = (): void => {
+      const tocDesktop: Element | null = document.querySelector('.theme-doc-toc-desktop');
+      if (tocDesktop) {
+        const isScrollable: boolean = tocDesktop.scrollHeight > tocDesktop.clientHeight;
+        const viewportHeight: number = window.innerHeight;
+        const tocRect = tocDesktop.getBoundingClientRect();
+        const spaceBelow: number = viewportHeight - tocRect.bottom;
+        const wouldOverflowWithTextArea: boolean = spaceBelow < 300;
+
+        setIsSidebarScrollable(isScrollable || wouldOverflowWithTextArea);
+
+      }
+    };
+    checkSidebarScrollable();
+
+    window.addEventListener('resize', checkSidebarScrollable);
+    const timer = setTimeout(checkSidebarScrollable, 100);
+
+    return (): void => {
+      window.removeEventListener('resize', checkSidebarScrollable);
+      clearTimeout(timer);
+    };
+  }, [location.pathname]);
+
+  const handleFeedbackClick = async (feedbackValue: string): Promise<void> => {
```

Comment:
Would it make sense to make the feedback value an enum instead of a string since there are two allowed values?